### ISSUE TITLE
Updating

### ISSUE THREAD

**Issue by Traqza** [nonsapiens/realaddressfactory#1]
Hello, are you willing to update the repository to most current laravel?

If you are unwilling to update it, will you allow me to assume the repository to maintain it going forward?

**Comment by stuartsebenza** [comment]
Hi there!  It ought to work on latest Laravel - can you confirm what issue you are encountering?

**Comment by ahathaway** [comment]
I would imagine it is identical or similar to this when trying to require the package from composer: 

```
$ sail composer require --dev nonsapiens/realaddressfactory
Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.10.128:9003 (through xdebug.client_host/xdebug.client_port).
./composer.json has been updated
Running composer update nonsapiens/realaddressfactory
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - geocoder-php/common-http[4.0.0, ..., 4.2.0] require php ^7.0 -> your php version (8.1.22) does not satisfy that requirement.
    - geocoder-php/google-maps-provider[4.0.0, ..., 4.4.0] require php ^7.0 -> your php version (8.1.22) does not satisfy that requirement.
    - geocoder-php/common-http 4.3.0 requires php ^7.2 -> your php version (8.1.22) does not satisfy that requirement.
    - geocoder-php/google-maps-provider 4.5.0 requires php ^7.2 -> your php version (8.1.22) does not satisfy that requirement.
    - geocoder-php/common-http[4.4.0, ..., 4.5.0] require psr/http-message ^1.0 -> found psr/http-message[1.0, 1.0.1, 1.1] but the package is fixed to 2.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - toin0u/geocoder-laravel[4.4.0, ..., 4.4.6] require illuminate/cache ^5.0|^6.0|^7.0|^8.0 -> found illuminate/cache[v5.0.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - toin0u/geocoder-laravel 4.4.2 requires geocoder-php/chain-provider dev-patch-1@dev -> found geocoder-php/chain-provider[dev-master, 4.0.0-beta1, ..., 4.5.0] but it does not match the constraint.
    - toin0u/geocoder-laravel[4.5.0, ..., 4.5.1] require illuminate/cache ^5.0|^6.0|^7.0|^8.0|^9.0 -> found illuminate/cache[v5.0.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - toin0u/geocoder-laravel 4.6.0 requires geocoder-php/google-maps-provider ^4.0 -> satisfiable by geocoder-php/google-maps-provider[4.0.0, ..., 4.7.1].
    - nonsapiens/realaddressfactory[v2.0.0, ..., v2.0.2] require toin0u/geocoder-laravel ^4.4 -> satisfiable by toin0u/geocoder-laravel[4.4.0, ..., 4.6.0].
    - geocoder-php/google-maps-provider[4.6.0, ..., 4.7.1] require geocoder-php/common-http ^4.0 -> satisfiable by geocoder-php/common-http[4.0.0, ..., 4.5.0].
    - Root composer.json requires nonsapiens/realaddressfactory * -> satisfiable by nonsapiens/realaddressfactory[v2.0.0, v2.0.1, v2.0.2].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require nonsapiens/realaddressfactory:*" to figure out if any version is installable, or "composer require nonsapiens/realaddressfactory:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by stuartsebenza** [comment]
Newest release supports Laravel 11 :-)
